sdk/communication/communication-common/review/communication-common.api.md

```diff
@@ -95,16 +95,15 @@ export const isPhoneNumberIdentifier: (identifier: CommunicationIdentifier) => i
 export const isUnknownIdentifier: (identifier: CommunicationIdentifier) => identifier is UnknownIdentifier;
 
 // @public
-export interface MicrosoftBotIdentifier {
-    botId: string;
-    cloud?: "public" | "dod" | "gcch";
-    isResourceAccountConfigured?: boolean;
+export interface MicrosoftTeamsAppIdentifier {
+    cloud?: "public";
```

Comment:
as this identifier cloud will be public always maybe we can get rid of this variable, wdyt @DominikMe?

Reply:
I agree, in that case we can remove the `cloud` property. It would still be possible to add it again later. Are there any plans to support this new identifier in sovereign clouds?

Reply:
Makes sense. Iv addressed and updates this PR as discussed in out chat